optuna/integration/botorch.py

```diff
@@ -508,6 +537,13 @@ def __init__(
         self._search_space = IntersectionSearchSpace()
         self._device = device or torch.device("cpu")
 
+        if consider_running_trials:
+            warnings.warn(
+                "``consider_running_trials`` option is an experimental feature."
+                " The interface can change in the future.",
+                ExperimentalWarning,
+            )
+
```

Comment:
`BoTorchSampler` is an experimental class already. I don't think we need this warning.